### DISCUSSION TITLE
renLUNA tokens stuck on Solana wormhole

### DISCUSSION THREAD

**Discussion by victoriousfighter** [wormhole-foundation/wormhole#223] [Problems / bugs] [answered]
Hello team,

My tokens got stuck on the solana wormhole

https://explorer.solana.com/address/7qEWvQ8T9vVWDCVxmpcntCok52dq4d9nwwsjpsvicd3n

^this is the trxn hash

In my ETH wallet, i have

0xec076720c5814ee7176b37661745467fff016030 these tokens instead

**Comment by hendrikhofstadt** [comment, marked as answer]
This is correct behaviour. The token on Ethereum is "wrapped renLUNA". The Wormhole cannot unwrap your renToken (which is another bridge). In order to unwrap them you need to send the tokens back to Solana and use the ren Bridge.

**Reply by misterbaka** [reply]
hi, i have a similar issue.  how would i send the wormhole wrapped tokens from ETH back to solana? On the wormhole UI i cant seem to get the wrapped token to show up.  thanks.

**Reply by leoluk** [reply]
You can create issues for wormholebridge.com here: https://github.com/solana-labs/oyster/issues
